apps/web/src/hooks/useConsent.ts

```diff
@@ -0,0 +1,17 @@
+import useLocalStorage from '@/services/local-storage/useLocalStorage'
+import { useCallback } from 'react'
+
+const useConsent = (storageKey: string) => {
```

Comment:
A very lightweight wrapper around useLocalStorage but it was easier than duplicating the previous `useStakeConsent` hook and we should be able to use the useLocalStorage hook instead of a custom one that hooks into the localStorage object.